src/documentation/markdown.rs

```diff
@@ -77,6 +84,7 @@ impl<'a> PlainOverlay<'a> {
         let mut inception = false;
         let mut skip_link_text = false;
         let mut skip_table_text = false;
+        let mut current_link_type: Option<LinkType> = None;
```

Comment:
I think we can find a better name, current does not really imply why this is used or what it's used for - at the very least add a descriptive comment.